test/models/albef/test_albef.py

```diff
@@ -37,8 +37,8 @@ def vision_encoder():
 
 
 @pytest.fixture(autouse=True)
```

Comment:
I was thinking of moving it but didn't want to potentially break the numerical tests, but I can go ahead and make that change

Reply:
It's currently setting seed like this (below). Since the fixture is autoused, the other fixtures should have a fresh seed as in the current implementation. That would be equivalent to pulling the seed as a separate fixture and turn on autouse and hopefully the downstream fixtures will still get the same random state.

https://github.com/facebookresearch/multimodal/blob/d9a3a2bd00cc5d40e7c288451f9455c52b0b7630/test/models/albef/test_albef.py#L25-L28